examples/tests/sdk_tests/bucket/load_test.test.w

```diff
@@ -2,8 +2,8 @@ bring cloud;
 
 let b = new cloud.Bucket();
 
-test "uploading many objects" {
+new std.Test(inflight () => {
   for i in 0..500 {
     b.put("test{i}", "{i}");
   }
-}
+}, timeout: 3m) as "uploading many objects";
```

Comment:
I wonder if we should open an issue to provide a way to pass these props in the current test syntax

Reply:
we already have an old one here: https://github.com/winglang/wing/issues/2594